examples/drupal_loadtest.rs

```diff
@@ -82,42 +82,42 @@ fn main() {
 
 /// View the front page.
 async fn drupal_loadtest_front_page(user: &GooseUser) {
-    let mut goose = user.get("/").await;
-
-    // Grab some static assets from the front page.
-    match goose.response {
-        Ok(r) => {
-            // Copy the headers so we have them for logging if there are errors.
-            let headers = &r.headers().clone();
-            match r.text().await {
-                Ok(t) => {
-                    let re = Regex::new(r#"src="(.*?)""#).unwrap();
-                    // Collect copy of URLs to run them async
-                    let mut urls = Vec::new();
-                    for url in re.captures_iter(&t) {
-                        if url[1].contains("/misc") || url[1].contains("/themes") {
-                            urls.push(url[1].to_string());
+    if let Ok(mut goose) = user.get("/").await {
```

Comment:
Let's use match.+ early return and not if let ... - please.

That makes the ? transition more natural.

Reply:
Some more justification:

Two reasons for early return:

a) More than two levels on indentation are by today’s standards usually considered a code smell

b) It’s good to have the same structure as used by ? operator later anyway. (It early returns on error).

Else you indent all, then outdent all for no reason and that makes the git blame way harder to read.

Reply:
Thanks for patiently explaining. I like it, implemented.